modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/eventsDefinitions.js

```diff
@@ -12,11 +12,10 @@
  * details.
  */
 
-export const ADD_TO_ORDER = 'refreshCartUsingData';
 export const AUTOCOMPLETE_VALUE_UPDATED = 'autocomplete-updated';
 export const CART_PRODUCT_QUANTITY_CHANGED = 'quantity-changed-from-cart';
-export const CHANGE_ACCOUNT = 'accountSelected';
-export const CHANGE_ORDER = 'orderSelected';
+export const CURRENT_ACCOUNT_UPDATED = 'current-account-updated';
```

Comment:
@garofalog These events are used in the MiniCart. Please change them there too, or it will blow up. :/